vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextArea.java

```diff
@@ -98,6 +115,7 @@ public TextArea(String label, String placeholder) {
      */
     public TextArea(String label, String initialValue, String placeholder) {
         this(label);
+        setLabel(label);
```

Comment:
Why? Label is assigned within `this(label)`

Reply:
Leftover.